### PR TITLE
Support lookup table source(use jdbc table source)

### PR DESCRIPTION
**Pull request by humengyu2012**

### CODE REVIEW

**Review by marsishandsome — APPROVED** [review]
LGTM

File: flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TiDBBaseDynamicTableSource.java [hunk 1]
```diff
@@ -55,4 +85,9 @@ protected String getRequiredProperties(String key) {
     return Preconditions.checkNotNull(properties.get(key), key + " can not be null");
   }
 
+  private long getPropertyLong(String key, long defaultValue) {
```
Comment (sunxiaoguang) [inline flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TiDBBaseDynamicTableSource.java:88]: Looks like this method is not being used at all

Comment (humengyu2012) [reply flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TiDBBaseDynamicTableSource.java:88]: Yes, I will delete this method.

**Pull request merged**